GameServer/Logic/Maps/SummonersRift.cs

```diff
@@ -333,11 +376,38 @@ public override void SetMinionStats(Minion minion)
                     minion.GetStats().Armor.BaseValue = 30.0f;
                     minion.GetStats().MagicResist.BaseValue = -30.0f;
                     minion.setMelee(true);
-                    minion.setAutoAttackDelay(5.9f / 30.0f); //Pretty sure this aint the true value
+                    minion.setAutoAttackDelay(15.0f / 30.0f);
                     break;
             }
         }
 
+        public void SpawnMinion(List<string> list, int minionNo, MinionSpawnPosition pos, List<Vector2> waypoints)
```

Comment:
If you insist on using strings instead of minion types in your lists, this should be done like this:
```CS
private const Dictionary<string, MinionSpawnType> _minionNameToType = new Dictionary<string, MinionSpawnType>
{
    {"MELEE", MinionSpawnType.MINION_TYPE_MELEE},
    {"CASTER", MinionSpawnType.MINION_TYPE_CASTER},
    {"CANNON", MinionSpawnType.MINION_TYPE_CANNON},
    {"SUPER", MinionSpawnType.MINION_TYPE_SUPER}
}

public void SpawnMinion(List<string> list, int minionNo, MinionSpawnPosition pos, List<Vector2> waypoints)
{
    if(list.Length <= minionNo)
    {
        return;
    }

    var minion = new Minion(_minionNameToType[list[minionNo]], pos, waypoints);
    AddObject(minion);
}
```

However, if you change your lists to use the minion types directly, there's no need for conversions, and you can simply just have
```CS
public void SpawnMinion(List<string> list, int minionNo, MinionSpawnPosition pos, List<Vector2> waypoints)
{
    if(list.Length <= minionNo)
    {
        return;
    }

    var minion = new Minion(list[minionNo], pos, waypoints);
    AddObject(minion);
}
```